docs/blog/csed-week/csed-week-2021.md

```diff
@@ -0,0 +1,51 @@
+# Computer Science Education Week 2021
+
+**Posted on November 8th, 2021 by [Jaqster](https://github.com/jaqster)**
+
+[Computer Science Education Week](www.https://csedweek.org) is coming up December 6th - 12th. This is an annual event to raise the visibility of computer science in education, and to inspire teachers and students to complete an "Hour of Code" during this week.
+
+For those of you already thinking about planning activities for CS Ed Week, here are some fun ideas using MakeCode!
+
+## MakeCode Arcade
+
+The theme for this year’s CS Ed Week is "CS is Everywhere" – highlighting how technology and computer science are helping us solve some of the world’s toughest problems.
+
+The 2021 MakeCode Arcade activity focuses on the impact that computing can have in savigfthe environment – and specifically how computers and technology can help combat forest fires. The **[Save the Forest!](https://arcade.makecode.com/hour-of-code-2021)** activity guides students through creating a game that will start and spread a forest fire, and then code their airtanker plane to spray water to put out the flames.
+
+![Forest fire game](/static/blog/csed-week/csed-week-2021/forest-fires.gif)
+
+There’s also an [Educator Guide](https://arcade.makecode.com/hour-of-code/educators-2021) which gives more information and guidance on how to run this activity in the classroom.
+
+### MakeCode Arcade Game Jam
+
+There’s nothing like a bit of competition to inspire people to code! In partnership with the [US Department of State](https://www.state.gov), we are running the first ever [Global MakeCode Arcade Game Jam](https://arcade.makecode.com/gamejam/global-2021) in December where students globally are invited to participate and show off the games they build using [MakeCode Arcade](https://arcade.makecode.com). The competition starts on November 29th when the theme of the game jam is announced. Submissions are due by December 17th when we’ll feature some of the best games submitted!
+
+![Game Jam banner](/static/blog/csed-week/csed-week-2021/game-jam-banner.png)
+
+Learn more at: https://arcade.makecode.com/gamejam/global-2021.
+
+## MakeCode in Minecraft
+
+[Minecraft: Education Edition](https://education.minecraft.net/en-us/homepage) is hosting another free coding world for anyone to download and play – you don’t need to buy a license or have an Office 365 account!
+
+![HoC 2021 splash banner](/static/blog/csed-week/csed-week-2021/hoc2021-splash-banner.jpg)
+
+This year’s Hour of Code experience is called "TimeCraft" where your player will travel back in time to fix **Time Splits** (mysterious errors in history) with the help of your Time Agent! There are 10 different coding challenges help save Jazz in the 1920’s, or to help change the shape of the Pyramids in Egypt from squares to triangles.
```

Comment:
This reads a bit odd... maybe change to, "There are 10 different coding challenges to fix these time splits.  For example, go back in time to help save Jazz in the 1920's, or help change the shape of the Pyramids in Egypt from squares back to triangles!"

Reply:
How about: "There are 10 different coding challenges to fix these time splits. In one of the challenges, the player goes back in time to help save Jazz in the 1920's. In another, the player helps to change the Pyramids in Egypt from square shapes back into triangles!"